templates/telecommunications/index.html

```diff
@@ -0,0 +1,207 @@
+{% extends "templates/one-column.html" %}
+
+{% block meta_description %}Companies around the world rely on Ubuntu for secure open source solutions. We work with our customers to meet the highest security standards.{% endblock %}
+
+{% block title %}Ubuntu solutions for telecommunications{% endblock %}
+
+{% block content %}
+<section class="p-strip--accent is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>Ubuntu for telcos</h1>
+      <p>As an established leader in cloud computing, Canonical let&rsquo;s you apply the webscale savings and best practice  that are necessary to succeed in the telco world. By using established open source software to automate the deployment and operation of NFVI and IoT solutions, we can help you become even more agile while reducing your costs and time to market.</p>
+      <p><a class="p-button--positive p-link--external" href="https://www.brighttalk.com/webcast/6793/290067" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Watch Cloud to Edge webinar', 'eventLabel' : 'Watch Cloud to Edge webinar', 'eventValue' : undefined });" >Watch Cloud to Edge webinar</a></p>
```

Comment:
Shouldn't there be a `Contact us` second CTA here.

Reply:
Added, requested link in copy doc